composer.json

```diff
@@ -30,11 +30,10 @@
         "doctrine/doctrine-fixtures-bundle": "~2.3",
         "phpunit/phpunit": "^4.8.35|^5.7|^6.1",
         "twig/twig": "~1.12|~2.0",
-        "nelmio/alice": "~1.7|~2.0",
+        "theofidry/alice-data-fixtures": "^1.0",
```

Comment:
you also need to dump php to `^7.1` (requirement of `theofidry/alice-data-fixtures`)

Reply:
I don't know, this is why I'm talking about a new major on the thread.

Indeed, PHP 7.1 is required for this dependency, but alice-data-fixtures is not on this bundle.

Reply:
If we retarget this PR to the 2.x, requiring 7.1 is already included.

Reply:
Rebased.